pkg/app/ssh.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2022 The envd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package app
+
+import (
+	"github.com/cockroachdb/errors"
+	"github.com/urfave/cli/v2"
+
+	"github.com/tensorchord/envd/pkg/home"
+	"github.com/tensorchord/envd/pkg/ssh"
+	sshconfig "github.com/tensorchord/envd/pkg/ssh/config"
+)
+
+var CommandSSH = &cli.Command{
+	Name:     "ssh",
```

Comment:
Is this what we planned for `envd attach`? `attach` sounds better to me

Reply:
I am not sure if we need this. the CLI UX design should be refined soon. Let's refine it later.